src/policy/fees.cpp

```diff
@@ -998,19 +998,34 @@ void CBlockPolicyEstimator::FlushUnconfirmed() {
     LogPrint(BCLog::ESTIMATEFEE, "Recorded %u unconfirmed txs from mempool in %gs\n", num_entries, (endclear - startclear)*0.000001);
 }
 
-FeeFilterRounder::FeeFilterRounder(const CFeeRate& minIncrementalFee)
+static std::set<double> MakeFeeSet(const CFeeRate& min_incremental_fee,
+                                   double max_filter_feerate,
+                                   double fee_filter_spacing)
 {
-    CAmount minFeeLimit = std::max(CAmount(1), minIncrementalFee.GetFeePerK() / 2);
+    std::set<double> feeset;
+
+    CAmount min_fee_limit = std::max(CAmount(1), min_incremental_fee.GetFeePerK() / 2);
```

Comment:
ecb5e10247a80356e1d22c2970e8c9b8fabff486
```suggestion
    const CAmount min_fee_limit{std::max(CAmount(1), min_incremental_fee.GetFeePerK() / 2)};
```